esp/public/media/scripts/program/modules/scheduling_checks.jsx

```diff
@@ -80,11 +93,32 @@ var SchedulingCheck = React.createClass({
     } else if (!this.state.data) {
       body = <div className="placeholder">loading...</div>;
     } else {
+      var data = JSON.parse(this.state.data); // Might not work on old browsers
+      var table;
+      if (data.headings.length == 0) {
+        var settings = {
```

Comment:
I'd just inline this in the `<SelectTable>`, I think that makes it easier to see what's happening.  I'd also recommend just passing a single header attribute, instead of a settings object, since there is just one setting.  If the number later becomes unmanageable, you can cross that bridge then.